stl/inc/atomic

```diff
@@ -625,36 +671,36 @@ struct _Atomic_storage<_Ty, 1> { // lock-free using 1-byte intrinsics
         }
     }
 
-    _NODISCARD _Ty load() const noexcept { // load with sequential consistency
+    _NODISCARD _TVal load() const noexcept { // load with sequential consistency
         const auto _Mem = _Atomic_address_as<char>(_Storage);
         char _As_bytes  = __iso_volatile_load8(_Mem);
         _Compiler_or_memory_barrier();
-        return reinterpret_cast<_Ty&>(_As_bytes);
+        return reinterpret_cast<_TVal&>(_As_bytes);
     }
 
-    _NODISCARD _Ty load(const memory_order _Order) const noexcept { // load with given memory order
+    _NODISCARD _TVal load(const memory_order _Order) const noexcept { // load with given memory order
         const auto _Mem = _Atomic_address_as<char>(_Storage);
         char _As_bytes  = __iso_volatile_load8(_Mem);
         _Load_barrier(_Order);
-        return reinterpret_cast<_Ty&>(_As_bytes);
+        return reinterpret_cast<_TVal&>(_As_bytes);
     }
 
-    _Ty exchange(const _Ty _Value, const memory_order _Order = memory_order_seq_cst) noexcept {
+    _TVal exchange(const _TVal _Value, const memory_order _Order = memory_order_seq_cst) noexcept {
         // exchange with given memory order
         char _As_bytes;
         _ATOMIC_CHOOSE_INTRINSIC(_Order, _As_bytes, _InterlockedExchange8, _Atomic_address_as<char>(_Storage),
             _Atomic_reinterpret_as<char>(_Value));
         return reinterpret_cast<_Ty&>(_As_bytes);

```

Comment:
In `load()`, `reinterpret_cast<_Ty&>(_As_bytes)` was changed to `reinterpret_cast<_TVal&>(_As_bytes)`, but not here.

Reply:
Yes, missed this one, thanks